lib/jsonld.js

```diff
@@ -934,6 +995,20 @@ jsonld.documentLoaders = {};
 jsonld.documentLoaders.node = require('./documentLoaders/node');
 jsonld.documentLoaders.xhr = require('./documentLoaders/xhr');
 
+// Optional DOM parser
+try {
+  jsonld.domParser = require('xmldom').DOMParser;
```

Comment:
This will support xmldom, if it's loaded. Are there other parsers that should be included? How to configure and test?